### PR TITLE
Adding VideoFxProcessor Guide

### PR DESCRIPTION
**Pull request by hungns2010**
**Issue #:** 
Add VideoFxProcessor Guide. 
**Description of changes:**
Adding VideoFxProcessor Guide
**Testing:**
Visually check it
*Can these tested using a demo application? Please provide reproducible step-by-step instructions.*
N/A

**Checklist:**

1. Have you successfully run `npm run build:release` locally?
yes

2. Do you add, modify, or delete public API definitions? If yes, has that been reviewed and approved?
no

3. Do you change the wire protocol, e.g. the request method? If yes, has that been reviewed and approved?
no

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: README.md
Comment (simmkyu) [inline README.md:100]: The NPM's page will have an incorrect link if you update the link in README.md after the release. Can you keep the same link?

File: guides/21_Background_Filter_VideoFX_Processor.md
Comment (ltrung) [inline guides/21_Background_Filter_VideoFX_Processor.md:64]: Should we clarify here that Android is currently not supported and if they need Android support please use the older ones?

File: guides/15_Background_Filter_Video_Processor.md
Comment (ltrung) [inline guides/15_Background_Filter_Video_Processor.md:2]: Please mention the Android support here.

File: guides/21_Background_Filter_VideoFX_Processor.md
Comment (ltrung) [inline guides/21_Background_Filter_VideoFX_Processor.md:189]: Shouldn't this section be below or the same as creating a videoFXConfig object?

File: guides/21_Background_Filter_VideoFX_Processor.md
Comment (ltrung) [inline guides/21_Background_Filter_VideoFX_Processor.md:252]: maybe a section here to start/stop video input?

File: CHANGELOG.md
Comment (simmkyu) [inline CHANGELOG.md:23]: nit: Do not change the entries from the previous version. Since you will merge this PR after publishing 3.13 to NPM, add your entry under `[3.14.0]`.

File: README.md
Comment (hungns2010) [reply README.md:100]: What do you mean? this is just point to the new link which only available after the release right?

File: guides/21_Background_Filter_VideoFX_Processor.md
Comment (hungns2010) [reply guides/21_Background_Filter_VideoFX_Processor.md:189]: it is in the same level as creating a `videoFxConfig` object. However you need to create a `videoFxConfig` first, then creating a `VideoFxProcessor`

File: guides/21_Background_Filter_VideoFX_Processor.md
Comment (hungns2010) [reply guides/21_Background_Filter_VideoFX_Processor.md:252]: I see the current guide for `Background_Filter_Video_Processor` also doesn't have those steps so I think it is not in this guide scope?

File: guides/15_Background_Filter_Video_Processor.md
Comment (hungns2010) [reply guides/15_Background_Filter_Video_Processor.md:2]: Asking my manager to confirm the change. WIll update after that.

File: guides/21_Background_Filter_VideoFX_Processor.md
Comment (hungns2010) [reply guides/21_Background_Filter_VideoFX_Processor.md:64]: Asking my manager to confirm the change. WIll update after that.

Comment (hungns2010) [reply guides/21_Background_Filter_VideoFX_Processor.md:64]: Fixed!

File: CHANGELOG.md
Comment (hungns2010) [reply CHANGELOG.md:23]: Sure we will remove this change. Thanks

File: guides/15_Background_Filter_Video_Processor.md
Comment (hungns2010) [reply guides/15_Background_Filter_Video_Processor.md:2]: So we need approval from PM (i.e. Chari) for this. Since we clearly mentioned about not supporting Android devices on the new guide, I think it is ok to not mention it here.

File: README.md
Comment (hungns2010) [reply README.md:100]: Discuss offline. This is fine to keep!

File: guides/21_Background_Filter_VideoFX_Processor.md
Comment (ltrung) [reply guides/21_Background_Filter_VideoFX_Processor.md:191]: I mean you would want to explain all the config properties in the above section `Creating a VideoFxConfig`. Not sure why we have a section about creating a video Fx config without explaining any of props then explain them 2 sections later.

**Review by ltrung — COMMENTED** [review]
Seem like you need CHANGELOG :D

File: guides/21_Background_Filter_VideoFX_Processor.md
Comment (hungns2010) [reply guides/21_Background_Filter_VideoFX_Processor.md:129]: So the way it works is, you create processor first, then create a video transform device by passing those processors. Like this: ```const proc = await this.videoFilterToProcessor(this.selectedVideoFilterItem);
    this.chosenVideoFilter = this.selectedVideoFilterItem;
    this.chosenVideoTransformDevice = new DefaultVideoTransformDevice(
        this.meetingLogger,
        innerDevice,
        [proc]
    );
    return this.chosenVideoTransformDevice;```

File: guides/15_Background_Filter_Video_Processor.md
Comment (hungns2010) [reply guides/15_Background_Filter_Video_Processor.md]: Discussed this offline!

File: guides/21_Background_Filter_VideoFX_Processor.md
Comment (hungns2010) [reply guides/21_Background_Filter_VideoFX_Processor.md]: Yeah that makes sense. Updated it!

**Comment by hungns2010** [comment]
> Seem like you need CHANGELOG :D

Add it now

File: CHANGELOG.md
Comment (ltrung) [inline CHANGELOG.md:12]: What is the official name here? Is it 2.0?

Comment (hungns2010) [reply CHANGELOG.md:12]: yeah it is the office name

**Review by ltrung — DISMISSED** [review]

**Review by ltrung — APPROVED** [review]

File: guides/15_Background_Filter_Video_Processor.md
Comment (michhyun1) [inline guides/15_Background_Filter_Video_Processor.md:2]: Shall we call it Version 2?

File: guides/21_Background_Filter_VideoFX_Processor.md
Comment (michhyun1) [inline guides/21_Background_Filter_VideoFX_Processor.md:5]: should we specify when you call the `.create()` function?

File: guides/21_Background_Filter_VideoFX_Processor.md
Comment (michhyun1) [inline guides/21_Background_Filter_VideoFX_Processor.md:7]: Amazon Chime SDK for Javascript  browser demo?

File: guides/21_Background_Filter_VideoFX_Processor.md
Comment (michhyun1) [inline guides/21_Background_Filter_VideoFX_Processor.md:52]: `An Amazon content delivery network loads the machine\-learning\-model files for background filters at runtime when the `.create()` function is invoked.`

File: guides/21_Background_Filter_VideoFX_Processor.md
Comment (michhyun1) [inline guides/21_Background_Filter_VideoFX_Processor.md:115]: mention the `isSupported` function?

File: guides/21_Background_Filter_VideoFX_Processor.md
Comment (michhyun1) [inline guides/21_Background_Filter_VideoFX_Processor.md:117]: mention the actual funciton name: .create()

**Pull request merged**
